### ISSUE TITLE
Question: why nemo and pyannote?

### ISSUE THREAD

**Issue by caryknoop** [mirix/approaches-to-diarisation#2]
Out of curiosity why use both? 

Pyannote can get speaker info, did you conclude it was insufficient or what was your motivation?  Just curious. :)

My problem with Nemo is that it is simply way too finicky for my liking, as soon as you integrate it with other software it breaks.  I have long given up on Nemo.

**Comment by mirix** [comment]
Hi,

Nemo is a dependency of the TitaNet models. I use Nemo just for computing the TitaNet voice embeddings for user identification. Before I was using speechbrain's ECAPA-TDNN embeddings. The later approach is simpler in terms of installation and maintenance, but I found TitaNet's to be more reliable.

I was using Pyannote to split the sound tracks in voice segments, but not any more. The latest script does not directly rely on pyannote. 

However, according to some reports, I have not verified this yet, pyannote is a Nemo dependency. 

In summary TitaNet embeddings enable a more precise speaker identification at the cost of complicating the installation of the dependencies.

**Comment by mirix** [comment]
The new pipeline:

https://github.com/mirix/approaches-to-diarisation/issues/10#issue-1873161024
